.drone.yml

```diff
@@ -1,13 +1,25 @@
+workspace:
+  base: /go
+  path: src/github.com/kowala-tech/kUSD
+
 pipeline:
 
-  test-and-build:
+  test:
+    image: kowalatech/go:1.0.1
```

Comment:
Why not `golang:1.9.2-alpine`? I think the only difference is that ours have supervisor, but I don't think we need it here

Reply:
your right, don't need that gometalinter from base box